vcd/resource_vcd_vapp_vm_hot_updates_test.go

```diff
@@ -0,0 +1,223 @@
+// +build vapp vm ALL functional
+
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"regexp"
+	"strings"
+	"testing"
+)
+
+func TestAccVcdVAppHotUpdateVm(t *testing.T) {
+	var (
+		vapp        govcd.VApp
+		vm          govcd.VM
+		hotVappName string = t.Name()
+		hotVmName1  string = t.Name() + "VM"
+	)
+
+	if testConfig.Media.MediaName == "" {
+		fmt.Println("Warning: `MediaName` is not configured: boot image won't be tested.")
+	}
+
+	var params = StringMap{
+		"Org":         testConfig.VCD.Org,
+		"Vdc":         testConfig.VCD.Vdc,
+		"EdgeGateway": testConfig.Networking.EdgeGateway,
+		"Catalog":     testSuiteCatalogName,
+		"CatalogItem": testSuiteCatalogOVAItem,
+		"VAppName":    hotVappName,
+		"VMName":      hotVmName1,
+		"Tags":        "vapp vm",
+		"Media":       testConfig.Media.MediaName,
+	}
+
+	configTextVM := templateFill(testAccCheckVcdVAppHotUpdateVm, params)
+
+	params["FuncName"] = t.Name() + "-step1"
+	configTextVMUpdateStep1 := templateFill(testAccCheckVcdVAppHotUpdateVmStep1, params)
+
+	params["FuncName"] = t.Name() + "-step2"
+	configTextVMUpdateStep2 := templateFill(testAccCheckVcdVAppHotUpdateVmStep2, params)
+
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+
+	debugPrintf("#[DEBUG] CONFIGURATION: %s\n", configTextVM)
```

Comment:
please add a print for the other configuration texts

Reply:
added